09-features.md

```diff
@@ -39,6 +39,7 @@ The Context column decodes as follows:
 | 16/17 | `basic_mpp`                      | Node can receive basic multi-part payments                | IN9      | `payment_secret`  | [BOLT #4][bolt04-mpp]                 |
 | 18/19 | `option_support_large_channel`   | Can create large channels                                 | IN       |                   | [BOLT #2](02-peer-protocol.md#the-open_channel-message) |
 | 20/21 | `option_anchor_outputs`          | Anchor outputs                                            | IN       | `option_static_remotekey` | [BOLT #3](03-transactions.md)         |
+| 22/23 | `option_zero_fee_second_stage`   | Anchor commitment type with zero fee second-stage HTLC txs| IN       | `option_anchor_outputs` | [BOLT #3](03-transactions.md)         |
```

Comment:
What about adding the original vulnerability mail as a footnote, let's ease understanding of this cryptic option rational by future LN devs ?

Reply:
Good idea, added!